site.ts

```diff
@@ -141,18 +148,22 @@ export function inject({ config, posthog }) {
     }
 
     const createSurveyPopup = (survey) => {
+        const surveyQuestionType = survey.questions[0].type
```

Comment:
we'll make this better when we implement multi step surveys

Reply:
I think it's worth defining a schema right now so that we don't make another breaking change in the future, given that we'll release surveys for general use before multi step surveys.

I'd propose: Everything making into the survey is in the questions array: be it just text, or anything that requires a response.

Then, the event we capture has properties of the form:

`$survey_question{index}` and `$survey_response{index}` where index is from `1 to len(questions)-1` and the 0th index has no index suffix.

So for example, if I had three questions in the survey:

```
survey.questions = [
{
    type: "open",
    question: "question 1?",
},
{
    type: "multiple_choice_single",
    question: "question 2?",
    choices: ["a", "b", "c", "d"],
},
{
    type: "nps",
    question: "rate me?",
    choices: [1,2,3,4,5],
}
]
```

and someone responds to them with `"whazzaap?", "c", 4` ; then the captured event should look like:


```
posthog.capture('survey sent', {
        $survey_name: survey.name,
        $survey_id: survey.id,
        $survey_question: "question 1?",
        $survey_response: "whazzaap?",
        $survey_question1: "question 2?",
        $survey_response1: "c",
        $survey_question2: "rate me?",
        $survey_response2: 4,
        sessionRecordingUrl: sessionRecordingUrl,
    })
```

Reply:
If you disagree, worth hashing this out now before we merge :)

Reply:
yaaa that sounds like a good plan